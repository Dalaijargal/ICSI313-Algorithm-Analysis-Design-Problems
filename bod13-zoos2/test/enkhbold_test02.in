100000 10
111 222 333 444 555 666 777 888 999 13