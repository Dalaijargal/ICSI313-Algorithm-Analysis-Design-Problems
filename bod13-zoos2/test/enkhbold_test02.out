278079972653492